src/codeu/chat/util/CommandTokenizer.java

```diff
@@ -0,0 +1,68 @@
+package codeu.chat.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+final public class CommandTokenizer {
+
+	private String source;
```

Comment:
Similarly, comments explaining what these variables are used for would be helpful.